### PR TITLE
feat: Add WKID primary inputs to CreateWorkOrderInputs, CreateServiceRequest & CreateInspection

### PR DESCRIPTION
**Pull request by ggfevans**
Added WKID to CreateWorkOrderInputs

### CODE REVIEW

**Comment by ggfevans** [comment]
Closes #9

**Comment by rcooney** [comment]
We should make this same change to `CreateServiceRequest.ts` and `CreateInspection.ts`.

**Comment by rcooney** [comment]
To fix the "Lint PR" check you'll need to change your PR title to be prefixed with "feat: ".

File: src/activities/ams/CreateWorkOrder.ts
Comment (ggfevans) [inline src/activities/ams/CreateWorkOrder.ts:1]: Cleaned up syntax of changes, reverted UnsupportedApps change

**Review by rcooney — APPROVED** [review]

**Pull request merged**